mgmt_cli_test.py

```diff
@@ -506,6 +512,86 @@ def test_backup_rate_limit(self):
         self.verify_backup_success(mgr_cluster=mgr_cluster, backup_task=backup_task)
         self.log.info('finishing test_backup_rate_limit')
 
+    @staticmethod
+    def _get_all_snapshot_files_s3(cluster_id, bucket_name, region_name):
+        file_set = set()
+        try:
+            s3_client = boto3.client('s3', region_name=region_name)
+            paginator = s3_client.get_paginator('list_objects')
+            pages = paginator.paginate(Bucket=bucket_name, Prefix=f'backup/sst/cluster/{cluster_id}')
+            for page in pages:
+                content_list = page["Contents"]
+                file_set.update([item["Key"] for item in content_list])
+        except KeyError as err:
```

Comment:
why not avoiding this exception by checking in advance: 
`if "Contents" in page`